CHANGELOG.md

```diff
@@ -4,109 +4,110 @@
+- Updated how [Node Transport Options are passed down](./MIGRATION.md#node-transport-changes).
+- Start propogating [`baggage` HTTP header](https://www.w3.org/TR/baggage/) alongside `sentry-trace` header to [propogate additional tracing related information.](./MIGRATION.md#propagation-of-baggage-header). (#5133)
+- Renamed `registerRequestInstrumentation` export to `instrumentOutgoingRequests` in `@sentry/tracing`. (#4859)
+- Renamed `UserAgent` integration to `HttpContext`. (#5027)
+- Replaced `BrowserTracing` integration's `maxTransactionDuration` option with `finalTimeout` option in the `@sentry/tracing` package and reset `idleTimeout` based on activities count. This should improve accuracy of web-vitals like LCP by 20-30%. (#5044)
+- [Updated `@sentry/angular` to be compiled by the angular compiler](./MIGRATION.md#sentry-angular-sdk-changes). (#4641)
+- Made tracing package treeshakable (#5166)
+
+- Removed support for [Node v6](./MIGRATION.md#dropping-support-for-nodejs-v6). (#4851)
+- Removed `@sentry/minimal` package in favour of using [`@sentry/hub`](./MIGRATION.md#removal-of-sentryminimal). (#4971)
+- Removed support for Opera browser pre v15 (#4923)
+- Removed `ignoreSentryErrors` option from AWS lambda SDK. Errors originating from the SDK will now *always* be caught internally. (#4994)
+- Removed `Integrations.BrowserTracing` export from `@sentry/nextjs`. Please import `BrowserTracing` from `@sentry/nextjs` directly.
+- Removed static `id` property from `BrowserTracing` integration.
+- Removed `SDK_NAME` export from `@sentry/browser`, `@sentry/node`, `@sentry/tracing` and `@sentry/vue` packages. (#5040)
+- Removed `Angular`, `Ember`, and `Vue` integrations from `@sentry/integrations` [in favour of the explicit framework packages: `@sentry/angular`, `@sentry/ember`, and `@sentry/vue`](./MIGRATION.md#removal-of-old-platform-integrations-from-sentryintegrations-package). (#4893)
+- Removed [enums `Status`, `RequestSessionStatus`, and `SessionStatus`.](./MIGRATION.md#removed-enums). Deprecated [enums `SpanStatus` and `Severity`](./MIGRATION.md#deprecated-enums). This was done to save on bundle size. (#4891) (#4889) (#4890)
+- Removed support for deprecated `@sentry/apm` package. (#4845)
+- Removed deprecated `user` field from DSN interface. `publicKey` should be used instead. (#4864)
+- Removed deprecated `getActiveDomain` method and `DomainAsCarrier` type from `@sentry/hub`. (#4858)
+- Removed `eventStatusFromHttpCode` to save on bundle size.
+- Removed usage of deprecated `event.stacktrace` field. (#4885)
+- Removed raven-node backward-compat code (#4942)
+- Removed `showReportDialog` method on `BrowserClient` (#4973)
+- Removed deprecated `startSpan` and `child` methods (#4849)
+- Removed deprecated `frameContextLines` options (#4884)
+- Removed `Sentry` from window in the Gatsby SDK (#4857)
+
+#### Internal
```

Comment:
Considering most users won't care about this section, should we move it to the bottom?

Reply:
Yup can do